OpenSim/Analyses/JointReaction.h

```diff
@@ -72,12 +72,12 @@ OpenSim_DECLARE_CONCRETE_OBJECT(JointReaction, Analysis);
         /* Joint reference*/
         const Joint* joint;
         /* The body upon which the force is applied */
-        const PhysicalFrame* appliedOnBody;
+        const Frame* appliedOnBody;
```

Comment:
This should remain `PhysicalFrame` since the Joint cannot join non-physical frames. Should update the comment too.